### PR TITLE
sphinx-intl: init at 2.0.1

### PR DESCRIPTION
**Pull request by EuAndreh**
###### Motivation for this change

Add `sphinx-intl` package.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Execution of binary tested with:
```shell
nix-shell -I nixpkgs=. -p 'python3.withPackages (p: [ p.sphinx-intl ])' --run sphinx-intl
```

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -2705,6 +2705,10 @@
       fingerprint = "67FE 98F2 8C44 CF22 1828  E12F D57E FA62 5C9A 925F";
     }];
   };
+  euandreh = {
```
Comment (IvarWithoutBones) [inline maintainers/maintainer-list.nix:2708]: This should be in a seperate commit.

File: pkgs/development/python-modules/sphinx-intl/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{ lib
```
Comment (IvarWithoutBones) [inline pkgs/development/python-modules/sphinx-intl/default.nix:1]: I don't see `lib` being used anywhere.

**Review by IvarWithoutBones — CHANGES_REQUESTED** [review]
Just 2 minor nits, otherwise LGTM :+1: 

Also, merge conflict in `pkgs/top-level/python-packages.nix` should be fixed.

**Comment by EuAndreh** [comment]
Ivar <notifications@github.com> writes:

> @IvarWithoutBones requested changes on this pull request.
>
> Just 2 minor nits, otherwise LGTM :+1: 
>
>
>
>> @@ -2705,6 +2705,10 @@
>        fingerprint = "67FE 98F2 8C44 CF22 1828  E12F D57E FA62 5C9A 925F";
>      }];
>    };
> +  euandreh = {
>
> This should be in a seperate commit.

👍, I'll rebase and split the commits.

>> @@ -0,0 +1,27 @@
> +{ lib
>
> I don't see `lib` being used anywhere.

It's used on `with lib.maintainers;`.

**Comment by EuAndreh** [comment]
I've rebased with master and split the commits.

@IvarWithoutBones could you review again?

File: pkgs/development/python-modules/sphinx-intl/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    sha256 = "1d1q0sanjp4nkfvhsxi75zf3xjyyi8nzxvl3v7l0jy9ld70nwnmj";
+  };
+
+  propagatedBuildInputs = [ Babel click sphinx ];
+
+  meta = with stdenv.lib; {
+    description = "Translation support for Sphinx";
+    homepage = "https://github.com/sphinx-doc/sphinx-intl";
+    license = licenses.bsd2;
+    maintainers = with lib.maintainers; [ euandreh ];
```
Comment (IvarWithoutBones) [inline pkgs/development/python-modules/sphinx-intl/default.nix:25]: `lib` is not required here, as its already in scope due to the `with stdenv.lib;` above.
```suggestion
    maintainers = with maintainers; [ euandreh ];
```

**Comment by EuAndreh** [comment]
Ivar <notifications@github.com> writes:

> @IvarWithoutBones commented on this pull request.
>
>
>
>> +buildPythonPackage rec {
> +  pname = "sphinx-intl";
> +  version = "2.0.1";
> +
> +  src = fetchPypi {
> +    inherit pname version;
> +    sha256 = "1d1q0sanjp4nkfvhsxi75zf3xjyyi8nzxvl3v7l0jy9ld70nwnmj";
> +  };
> +
> +  propagatedBuildInputs = [ Babel click sphinx ];
> +
> +  meta = with stdenv.lib; {
> +    description = "Translation support for Sphinx";
> +    homepage = "https://github.com/sphinx-doc/sphinx-intl";
> +    license = licenses.bsd2;
> +    maintainers = with lib.maintainers; [ euandreh ];
>
> `lib` is not required here, as its already in scope due to the `with stdenv.lib;` above.

👍

I've rebased again.

File: pkgs/development/python-modules/sphinx-intl/default.nix [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+, buildPythonPackage
+, fetchPypi
+, Babel
+, click
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-intl";
+  version = "2.0.1";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/sphinx-intl/default.nix:11]: Please disable it before Python 3.5:

```
ERROR: Package 'sphinx-intl' requires a different Python: 2.7.18 not in '>=3.5'
```

File: pkgs/development/python-modules/sphinx-intl/default.nix [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1d1q0sanjp4nkfvhsxi75zf3xjyyi8nzxvl3v7l0jy9ld70nwnmj";
+  };
+
+  propagatedBuildInputs = [ Babel click sphinx ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/sphinx-intl/default.nix:18]: Please disable tests or fetch from GitHub where maybe tests are included:

```
writing manifest file 'sphinx_intl.egg-info/SOURCES.txt'
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 99544` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>python27Packages.sphinx-intl</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python37Packages.sphinx-intl</li>
    <li>python38Packages.sphinx-intl</li>
  </ul>
</details>

**Comment by EuAndreh** [comment]
I've disabled for older Python versions and rebased with master.

`nixpkgs-review pr 99544` runs successfully for me now.

File: pkgs/development/python-modules/sphinx-intl/default.nix (hunk 5 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/sphinx-intl/default.nix:20]:
```
# package does not contain any tests
doChecks = false;
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by EuAndreh** [comment]
Sandro <notifications@github.com> writes:

> # package does not contain any tests
> doChecks = false;

Done and rebased with master.

**Comment by EuAndreh** [comment]
BTW, thanks @SuperSandro2000 for the reviews!

File: maintainers/maintainer-list.nix [hunk 6]
```diff
@@ -2765,6 +2765,10 @@
...
+    name = "EuAndreh";
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:2769]: Please add your github name and id like the entries above and below you.

**Comment by SuperSandro2000** [comment]
> Done and rebased with master.

Please do not rebase on master. This is only necessary if there are eval errors. Otherwise I cannot follow the changes you did that easily.

**Comment by EuAndreh** [comment]
Closing stale pull request.  Future interested parties may pick this up and merge in the future.

**Pull request closed without merging**